WorkoutTest/ProgramViewController.swift

```diff
@@ -56,17 +52,42 @@ final class ProgramViewController: UIViewController {
       self?.myRefreshControl.endRefreshing()
     })
   }
+  
+  private func configureSearchBar() {
+    searchBar = UISearchBar()
+    searchBar.placeholder = "Search Programs"
+    searchBar.delegate = self
+    navigationItem.titleView = searchBar
+  }
 }
 
-extension ProgramViewController {
-  private func configureTableView() {
-    view.addSubview(tableView)
-    tableView.snp.makeConstraints {
-      $0.edges.equalToSuperview()
-    }
+//MARK: - configureCollectionViev + setupLayout
+
+private extension ProgramViewController {
+  func configureCollectionView() {
+    collectionView = UICollectionView(frame: view.bounds, collectionViewLayout: setupLayout())
```

Comment:
this is a bad way to setup a view. Better use `lazy var ` as it is done for `myRefreshControl` or `dataSource`